### PR TITLE
[expo-web-browser] Extract WebBrowser API to a universal module

### PR DESCRIPTION
**Pull request by sjchmiela**
`WebBrowser` screen of NCL works ok.

### CODE REVIEW

File: android/versioned-abis/expoview-abi26_0_0/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -42,6 +42,9 @@
           android:name="android.support.FILE_PROVIDER_PATHS"
           android:resource="@xml/provider_paths"/>
       </provider>
+        <activity
```
Comment (tsapeta) [inline android/versioned-abis/expoview-abi26_0_0/src/main/AndroidManifest.xml]: indentation?

File: android/versioned-abis/expoview-abi27_0_0/src/main/java/abi27_0_0/host/exp/exponent/modules/internal/ChromeTabsManagerActivity.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      finish();
+    }
+  }
+
+  @Override
+  protected void onNewIntent(Intent intent) {
+    super.onNewIntent(intent);
+    setIntent(intent);
+  }
+}
```
Comment (tsapeta) [inline android/versioned-abis/expoview-abi27_0_0/src/main/java/abi27_0_0/host/exp/exponent/modules/internal/ChromeTabsManagerActivity.java]: no newline ¯\_(ツ)_/¯

File: packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  // By setting the modal presentation style to OverFullScreen, we disable the "Swipe to dismiss"
+  // gesture that is causing a bug where sometimes `safariViewControllerDidFinish` is not called.
+  // There are bugs filed already about it on OpenRadar.
+  [safariVC setModalPresentationStyle: UIModalPresentationOverFullScreen];
+
+  // This is a hack to present the SafariViewController modally
+  UINavigationController *safariHackVC = [[UINavigationController alloc] initWithRootViewController:safariVC];
+  [safariHackVC setNavigationBarHidden:true animated:false];
+
+  UIViewController *currentViewController = [UIApplication sharedApplication].keyWindow.rootViewController;
```
Comment (tsapeta) [inline packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m]: is this better than currentViewController provided by EXUtils?

**Review by tsapeta — APPROVED** [review]
lgtm 💪

File: packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m (hunk 3 above)
Comment (sjchmiela) [reply packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m]: Yup, displaying on `currentViewController` freezes app.

**Pull request merged**